app/quests/tests.py

```diff
@@ -1,3 +1,45 @@
+from urllib.parse import urlencode
+
+from django.contrib.auth.models import User
 from django.test import TestCase
+from django.urls import reverse
+
+from dashboard.models import Profile
+from rest_framework import status
+
+CURRENT_USERNAME = "bot_dude"
+CURRENT_PASSWORD = 'mimamamemima'
+
+
+class QuestTest(TestCase):
+    def setUp(self):
+        self.current_user = User.objects.create(
+            password=CURRENT_PASSWORD, username=CURRENT_USERNAME)
+        Profile.objects.create(user=self.current_user, data={}, hide_profile=False, handle=CURRENT_USERNAME)
+
+    def test_new_quest_not_raise_exception_with_negative_seconds_to_responds(self):
+        """Test abs function on seconds to prevent set negative second to respond on quests questions"""
+        self.client.force_login(self.current_user)
+
+        data = urlencode({
+            'points': -10,
+            'seconds_to_respond[]': -10,
+            'question[]': ['one?']})
+        response = self.client.post(reverse('newquest'),
+                                    data,
+                                    content_type="application/x-www-form-urlencoded",
+                                    HTTP_USER_AGENT='none')
+
+        self.assertTrue(status.is_success(response.status_code))
+
+    def test_new_quests_should_redirect_to_login_when_no_user_is_logged(self):
+        """Test abs function on seconds to prevent set negative second to respond on quests questions"""
```

Comment:
I think this description needs to be updated

Reply:
@danlipert done!